NeoML/include/NeoML/Dnn/Optimization/Graph.h

```diff
@@ -48,10 +51,14 @@ struct CLayerOutput final {
 
 	CLayerOutput( TLayer* layer, int index ) : Layer( layer ), Index( index ) {}
 	CLayerOutput() = default;
+	void clear() { Layer = nullptr; Index = NotFound; }
```

Comment:
_codestyle_
Multiline -> out of class
public scope -> start with Capital letter

Reply:
fixed